CMake/StampBuild.cmake

```diff
@@ -0,0 +1,60 @@
+# These env variables are used for configuring Travis CI builds.
```

Comment:
Is this file copy-pasted?

Reply:
I deleted setting an environment variable to set CMAKE_BUILD_TYPE in favour of setting the variable directly from the command line.